src/e2e/puppeteer/__tests__/caret.ts

```diff
@@ -1,21 +1,22 @@
 import { KnownDevices } from 'puppeteer'
+import gestures from '../../../test-helpers/gestures'
 import click from '../helpers/click'
 import clickBullet from '../helpers/clickBullet'
 import clickThought from '../helpers/clickThought'
 import emulate from '../helpers/emulate'
 import getEditingText from '../helpers/getEditingText'
 import getSelection from '../helpers/getSelection'
+import keyboard from '../helpers/keyboard'
 import paste from '../helpers/paste'
 import press from '../helpers/press'
 import refresh from '../helpers/refresh'
-import testIfNotCI from '../helpers/testIfNotCI'
+import swipe from '../helpers/swipe'
 import waitForEditable from '../helpers/waitForEditable'
 import waitForHiddenEditable from '../helpers/waitForHiddenEditable'
-import waitForSelector from '../helpers/waitForSelector'
 import waitForThoughtExistInDb from '../helpers/waitForThoughtExistInDb'
 import waitUntil from '../helpers/waitUntil'
 
-vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+vi.setConfig({ testTimeout: 60000, hookTimeout: 20000 })
```

Comment:
Is increasing the timeout necessary, or just for debugging? I'd hope the test can complete in less than 20 seconds, otherwise our test suite is going to run very slowly.

Reply:
Yes, correct. It was originally implemented for debugging purposes. During earlier troubleshooting efforts involving multiple solution attempts, the tests were frequently timing out in the CI environment, which led me to initially increase the timeout duration.